Library/Homebrew/extend/fileutils.rb

```diff
@@ -121,11 +121,6 @@ def make(*args)
     end
   end
 
-  if method_defined?(:ruby)
-    # @private
-    alias old_ruby ruby
-  end
-
   # Run the `ruby` Homebrew is using rather than whatever is in the `PATH`.
   def ruby(*args)
```

Comment:
Seems like this isn't used anywhere. Should we deprecate this?

Reply:
Good idea!